common/coordinator.go

```diff
@@ -17,3 +17,6 @@ type Coordinator struct {
 	// URL of the coordinators API
 	URL string `meddler:"url"`
 }
+
+// CoordinatorsNetworkPort is the port used by coordinators for libp2p
+const CoordinatorsNetworkPort = "3598"
```

Comment:
Maybe this should be part of the config file. If so, I think we could tune the network creating multiple networks between nodes to share tx. I think this would be usefull if we want share tx between some specific nodes (to test a specific functionality for example)

Reply:
The reason for this to be hardcoded is that we read the URL/IP from whatever it's registered on the SC, if the port wasn't hardcoded we wouldn't know to which port is each coordinator listening

>  I think this would be usefull if we want share tx between some specific nodes (to test a specific functionality for example)

We can achieve that by tweaking other parameters such as `discoveryServiceTagBase` or topic name